src/wpephpcompat.php

```diff
@@ -190,14 +190,14 @@ public function start_test() {
 			$this->debug_log( 'Processing: ' . $directory->post_title );
 
 			// Add the plugin/theme name to the results.
-			$scan_results .= 'Name: ' . $directory->post_title . "\n\n";
+			$scan_results .= __( 'Name: ', 'php-compatibility-checker' ) . $directory->post_title . "\n\n";
```

Comment:
Here we're translating `Name: `, but in the JavaScript we're just doing `Name`. Could these be the same translation?